readme.md

```diff
@@ -5,18 +5,18 @@
 The Guardian's scala-based deployment system is designed to automate deploys by providing a web application that 
 performs and records deploys, as well as providing various integration points for automating deployment pipelines.
 
+## Documentation
+Riff-Raff's documentation is available in the application (under the Documentation menu) but can also be viewed under
+[riff-raff/public/docs](riff-raff/public/docs) in GitHub.
+
 ## Requirements
-Riff-Raff and Magenta have been built with the tools we use at the Guardian
+Riff-Raff has been built with the tools we use at the Guardian
 and you will find it easiest if you use a similar set of tools. Riff-Raff:
 
  - relies on artifacts and `riff-raff.yaml` files describing builds being in S3 buckets with the artifacts having paths of 
   the form `project-name/build-number`
  - uses the AWS SDK and [Prism](https://github.com/guardian/prism) to do resource discovery
- - stores configuration, history and logs in a PostgreSQL database and a handful of DynamoDB tables (the eventual aim is to ditch DynamoDB altogether)
```

Comment:
I don't think we're currently planning to remove DynamoDB and this feels like too much detail given the brevity of the readme overall